sdk/ml/azure-ai-ml/azure/ai/ml/automl/_automl_image.py

```diff
@@ -30,6 +30,10 @@ def _create_image_job(
     **kwargs,
 ):
     """Helper function to create objects for AutoML Image jobs."""
+
+    # Temporary fix for https://msdata.visualstudio.com/Vienna/_workitems/edit/2385143
+    kwargs["log_training_metrics"] = False
```

Comment:
should this be:

```suggestion
    kwargs["log_training_metrics"] = "Disabled"
```

instead?
Referring to the REST API here: https://github.com/Azure/azure-sdk-for-python/blob/ad455e4199ca15e490a030e8315e3a5c8f64ad33/sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2023_04_01_preview/models/_models_py3.py#L17072-L17075

Reply:
done.